docs/pages/blog/2023-mui-values.md

```diff
@@ -0,0 +1,242 @@
+
+Finally, we organized all of these cards into little clusters based on relatedness, and started to identify their underlying values and themes.
+
+We came up with a list of words that we felt best encapsulated each cluster. This became our values "shortlist."
+
+<img alt="description" src="/static/blog/2023-mui-values/figjam-board.png" width="1200" height="500" loading="lazy" />
+
+## Stage three: transforming the shortlist
+
+36.
+
+That's how many words surfaced in our workshop as potential values—or at least, as words to inspire the copywriting process.
+
+Narrowing down the shortlist gave us a chance to both find _and_ stress-test the strongest recurring themes. 🧪
+
+To do this, we immediately eliminated [what Patrick Lencioni calls "permission-to-play" values](https://www.youtube.com/watch?v=bNRn6FimzoU).
+These are the things as basic as showing respect, which might be _valuable_ to a great team culture and performance, but aren't exactly _exciting_ or _measurable_.
+They're simply the minimum required standard of behavior to work together.
+
+We also started striking out anything that could not clearly be expressed _as an action._
+
+It's [what Simon Sinek was talking about](https://www.youtube.com/watch?v=IQuYzXWXDqI) when he said "honesty is not a value."
+If there was not a clear and natural way to transform something into an action or provide examples of how it would be lived in our specific workplace, it was out. 🙅
+
+Success!
+
+We narrowed the list down from 36 to _just six._
+
+Now it was time to transform these six words into actionable phrases to inspire the very actions we want to see championed on a daily basis.
+
+## Stage four: pitching the 'PACT'
+
+All of the above and more influenced what's now referred to as our team "pact." (You'll see why.)
+
+But before we get there, I want to share a few examples of phrases that didn't wind up being a part of the pitch and why.
+
+Finding the right phraseology started with carefully reviewing _all_ team inputs.
+As I had a unique vantage point of all those inputs, I began to jot down phrases that effectively pulled from them _all_ and led with _action verbs._ 🕸️💪
+
+Things like "build in the open" made an appearance, but felt too limited.
+There was so much more we wanted to capture in the enduring theme of transparency, why it matters to us, and how we show it.
+
+Other phrases included "own your impact" and "maximize flexibility," but these, again, felt too limited and also generic.
+
+In short, they weren't right.
+
+Fortunately, this helped us find the ones that **\***are.**\*** 👏
+
+I presented a video presentation to our team, pitching this set of values and providing another opportunity for feedback and shaping.
+
+You can <a href="https://www.notion.so/mui-org/Values-behaviors-d3a1e1c60e2a4c0782f770cceada54bd?pvs=4#09dc19a420504a3090dc4fbe6a68a543">watch the "reveal" video for yourself here</a>! 🎬👀
```

Comment:
A quick note. `notion.so` requires to have a Notion account to be read. I changed this to `notion.site` in 48adec0fa7ea6b6aff051099966b2b17cedd1a98 so the content is easier to access for curious people 😄